website/src/content/docs/docs/standard-library/built-in-data-types.md

```diff
@@ -213,6 +255,45 @@ model PickProperties<Source, Keys>
 | Keys | The property keys to include. |
 
 
+#### Properties
+None
+
+### `Query` {#Query}
+
+A copy of the input model `T` with only the properties that are visible during the
+"Query" resource lifecycle phase.
+
+The "Query" lifecycle phase is used for properties passed as parameters to operations
+that read data, like HTTP GET or HEAD operations.
+
+This transformation is recursive, and will include only properties that have the
+`Lifecycle.Query` visibility modifier.
+
+If a `NameTemplate` is provided, the new model will be named according to the template.
+The template uses the same syntax as the `@friendlyName` decorator.
+```typespec
+model Query<T, NameTemplate>
+```
+
+#### Template Parameters
+| Name | Description |
+|------|-------------|
+| T | The model to transform. |
+| NameTemplate | The name template to use for the new model.<br /><br />* |
+
+#### Examples
+
+```typespec
+model Dog {
+  @visibility(Lifecycle.Read)
```

Comment:
Was this meant to be `LifeCycle.Query`?

Reply:
I improved these examples and gave some more thorough explanations. Query/Delete are probably rarely used in practice but do occur in Azure.